### ISSUE TITLE
Can't select other shipping method

### ISSUE THREAD

**Issue by KoenSlagter** [myparcelnl/magento#353]
When I have another shipping method enabled than just the MyParcel shipping methods, and try to select it in checkout, it won't deselect the MyParcel shipping methods, this is confusing.

**To Reproduce**
Have (multiple) MyParcel shipping methods enabled, and enable the default magento store pickup.
In checkout when you choose for the store pickup method, the MyParcel methods will stay selected. 

**Expected behavior**
I expect the MyParcel methods to be disabled when I select another shipping method

**Screenshots**
Screenshot: https://www.screencast.com/t/1nxctvc2gO

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chome
 - MyParcel Plugin version: 3.1.3

**Comment by Calypso971** [comment]
This problem has been reported several times before, and closed as being fixed, although upto 3.2.1-beta3 it wasn't. Trying out beta4 later today, but I don't have high hopes.

This problem was marked as important months and months ago. Please fix ASAP. And this time really fix.

**Comment by RichardPerdaan** [comment]
To solve this problem completely, we are busy with a complete new checkout. 

Because the checkout is used for all the MyParcel plugins, this is seen as a separate project. I can bless you with 100% certainty that it has been solved with this new checkout.

Multiple types of checkout will also be supported through this checkout. There are new techniques used like Vue.js. This will also make the code much more maintainable.

According to the schedule, we expect to have the first version available at the beginning of April.

**Comment by Kroellie** [comment]
Now I'm getting worried. Again a complete new checkout? What about 3.2.1 beta, will you finish it to a final product?

**Comment by RichardPerdaan** [comment]
The 3.2.1 beta will be finished to a final product. Only the checkout problem will not be solved.

What we are renewing has to do with a different repository ([myparcel/checkout](https://github.com/myparcelnl/checkout))

**Comment by Calypso971** [comment]
> Now I'm getting worried. Again a complete new checkout? What about 3.2.1 beta, will you finish it > to a final product?

What worries me is how long things take to get fixed and the lack of communication about it. Over a year ago (Jan 14, 2019) issue 228 was opened about this problem. First reaction from the developer: Mar 13 (2 months later). "I want to fix this issue". Then complete silence until Aug 21 "We're looking into a new checkout" and so it carries on without news until Oct 2 "It will be fixed in the next release". Not. And the issue has been closed at the same time. 

Pushed it a couple of times, no reaction. In the meanwhile issue 319 is there, Nov 11. Same issue again. Nov 28: Richard says it's fixed again and again closed the issue at the same moment. But it wasn't fixed so I reported it. No reaction. 

Then this issue; Jan 15. No reaction until today. 

To be honest, Richard, belief that the plugin can be used isn't there anymore. And thus we can't use MyParcel anymore. I've had too many complaints from customers facing this issue, and after over a year, saying that "a new checkout will fix it" won't do. This takes too long, both reacting to the issues as fixing it.

**Comment by RichardPerdaan** [comment]
Hi @Calypso971 

I am sorry to hear that you feel that way.
I think there have been improvements in the plugin in the past year. We have indeed been able to solve this problem regarding the checkout once, until Magento made a new release.

Because our plugin does not have to depend on the changes made by Magento, we have made the choice to change the architecture that was conceived three years ago. This is the reason that it takes longer for this is resolved.

Creating a new checkout is not a small project because we are currently working on improvements for functionalities and also in our API. This will also make the checkout 4 times faster.
